src/server/controllers/dockerController.ts

```diff
@@ -102,9 +102,11 @@ const dockerController = {
   },
 
   getImagesList: async () => {
+    console.log('images dockController')
     const docker = await new Docker({socketPath: '/var/run/docker.sock'})
     const list = await docker.listImages()
-    .then(list => { return list })
+    .then(list => { console.log(list); return list })
```

Comment:
can you remove this file from the commit? These are testing changes, not to be pushed to dev branch unless functionality is broken?